content/docs/procedures/_index.md

```diff
@@ -0,0 +1,4 @@
+This directory collects descriptions of various compiler-team "procedures". 
```

Comment:
I think files like these can be removed, there's not much value in having small almost-empty pages at the root of directories (there was when it was being browsed through GitHub's UI but not with a website).